src/gui/tray/wheelhandler.cpp

```diff
@@ -0,0 +1,280 @@
+/*
```

Comment:
Should we not pack that into the third party folder?

Reply:
If not, I would tend to say we should adjust the naming to make it more consistent with the current code base.

Reply:
Moved to the 3rdparty repo